recipe/patch_yaml/zlib.yaml

```diff
@@ -0,0 +1,9 @@
+# zlib was pinning to minor level, but 1.2 is compatible with 1.3
+if:
+  has_depends: libzlib >=1.2*
+  timestamp_gt: 1714521600000  # 2024-05-01 00:00 UTC
+  timestamp_lt: 1717200000000  # 2024-06-01 00:00 UTC
+then:
+  - loosen_depends:
+      name: libzlib
+      upper_bound: "1.4"
```

Comment:
We should change to `2`. Can you please open an issue in zlib-feedstock about moving to `x` isntead of `x.x`?

Reply:
I already have a branch, will open a PR as soon as I get through my rerendering struggles